gammapy/image/lists.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Please add the license comment line that's at the top of any file in the Gammapy to `gammapy/image/lists.py` and `gammapy/image/tests/test_lists.py`.